### ISSUE TITLE
All endpoints return 403

### ISSUE THREAD

**Issue by Spaceface16518** [pixelfed/pixelfed#3051]
I've liked pixelfed for a while, so I decided to try and run a self-hosted, non-federated instance for my extended family. I have used the installation guide to set up pixelfed on a Raspberry Pi running Ubuntu Server using PostgresQL and Redis behind a Caddy server, supervising everything with systemd.

Everything seems to be working properly; there are no actual errors logged from php-fpm or caddy. However, when I try to access the endpoint, it returns 403 Forbidden instead of the login page or something. I cannot find any logs of errors besides caddy logging that a 403 was sent back to the user (without explanation).

Is this a common issue with pixelfed and/or php? I have experience writing and setting up self-hosted services, I never used php before, so I don't what to modify to fix this error. The permissions of the static/php files match those in the guide, and I am using tcp for both caddy-php and php-redis communication for simplicity.

Any help would be appreciated. Even just knowing where to find error logs would be very helpful. Thanks!

**Comment by samuelroland** [comment]
Hey, I think this SO post could help: https://stackoverflow.com/questions/37535315/where-are-logs-located

**Comment by Spaceface16518** [comment]
Oh thank you. I have switched `APP_DEBUG` to `true`. I have checked the php-fpm and caddy logs, as the SO post suggested, but didn't find anything mentioning errors (just my web server returning a 403). I didn't realize the `laravel.log` file existed, but it doesn't have anything new, just a database permissions error from a while ago (which was fixed). I've tried sending requests and restarting the php-fpm and pixelfed task scheduler services, but nothing was added to any of the logs.
